packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -124,7 +124,7 @@ export class Governance {
    *
    * @param {VoteData} data Vote data
    * @param {string} data.proposalId Proposal id
-   * @param {number} data.masternodeId Masternode id
+   * @param {number} data.masternodeId Masternode id/owner address/operator address
```

Comment:
I wanted to a suggest a different variable name to better represent this param, but i cant think of any 😅